src/store/search.js

```diff
@@ -81,7 +82,7 @@ export const mediaSpecificFilters = {
 }
 
 /** @type {import('./types').Filters} */
-export const filterData = {
+export const filterData = deepFreeze({
```

Comment:
Adding this because (a) we treat this data as immutable and (b) this gives us assurances that it isn't being inadvertently changed.

Reply:
The problem here is the provider filters. They are set after receiving a response from the API with all the provider data, and in the search state initialization they are blank.

Reply:
I'm not seeing any issues related to setting providers, either client-side in the filters or when setting them by 'default' in the ssr-ed url.